cmd/workers/workers.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"net/http"
+	"sync"
+	"time"
+)
+
+var client = &http.Client{}
+var Wg sync.WaitGroup
+
+func worker(id int, itr int, ch chan<- string) {
+	defer Wg.Done()
+	for i := 0; i < itr; i++ {
+		randomWaitTime := time.Duration(rand.Int63n(int64(time.Second)))
+
+		time.Sleep(randomWaitTime)
+		req, err := http.NewRequest("GET", "http://localhost:8080/health", nil)
```

Comment:
I called only 1 endpoint here, imagine calling all of our endpoints and millions of requests at 1 end point would be insane

Reply:
🤯  
#ddos machine 😆